data_adapter_oemof/calculations.py

```diff
@@ -214,10 +214,16 @@ def handle_nans(group_df: pd.DataFrame) -> pd.DataFrame:
         if column in ["method", "source", "comment", "bandwidth_type"]:
             continue
 
-        if group_df[column].nunique(dropna=False) > 1:
+        if group_df.isna().sum>0:
             if column in max:
                 group_df[column].fillna(max_value, inplace=True)
             elif column in min:
                 group_df[column].fillna(min_value, inplace=True)
             else:
-                group_df[column].fillna(min_value, inplace=True)
+                if "type" in group_df.columns:
+
+                    raise ValueError(f"In column {column} for process {group_df['type']} nan values are found"
+                                     f"please make sure to only provide complete datasets")
+                else:
+                    raise ValueError(f"In column {column} nan values are found"
+                                     f"please make sure to only provide complete datasets")
```

Comment:
This could cause us more trouble then letting None in there IMO.
So maybe not check for it instead...

Reply:
Yes it would but yet I am facing this issue again now [with x2x dataset](https://openenergyplatform.org/dataedit/view/model_draft/x2x_delivery_hydrogen_pipeline_new_1) and it is hard to guess which dataset is faulty. 

Having an Error here or in `data_adapter` would help since it is actually an error? We should not ignore and detect it somewhere (as early as possible)?

Reply:
> Having an Error here or in `data_adapter` would help since it is actually an error? We should not ignore and detect it somewhere (as early as possible)?

But I think there will be many cases in datasets where this is the case - and I don't see people fixing datasets quick enough. So better be prepared for None values and ignore them as far as possible (tabular sets default values in most cases)?

Reply:
Tabular will not be able to set default values since it is impossible to know which values are missing. We are loosing information if single entries are missing because tabular is receiving a list for changing values. If the list is shorter than the amount of periods we will not know anymore which years have gone missing. 

Maybe we can adjust the reading of datapackage in tabular to use pass a nan if a string is (eg. "nan") is found in such a list?